docs/android/gradle.md

```diff
@@ -1,221 +1,230 @@
 
 # Gradle Build Instructions - Android
 
-These instructions guide you through the process of integrating Product Science instrumentation into an Android application's Gradle build process.
+Follow these steps to instrument your Java/Kotlin Android app with PS Plugin. Once you have successfully built, installed and run the app, you will be able to record and analyze traces.
+
+If you get stuck, please refer to our [FAQ](https://docs.productscience.app/android/integration-faq-android/) for additional steps and guidance, or reach out to customer support.
 
 !!! info
-    If your build environment does not allow network access to our servers `https://prod.productscience.app/api/v1/*`, please add to your allowlist.
-    If your network settings prevent adding this endpoint, you will be provided with plugin and config archives detailed in sections below.
+   If your build environment does not allow network access to our [servers](https://prod.productscience.app/api/v1/*), please add to your allowlist. If your network settings prevent adding this endpoint,      please reach out to our sales team or customer support.
```

Comment:
@gmoshonov please don't make long lines

Link here should not be clickable. When [..])(...) syntax is used client can't see the URL pattern itself to add it to allowlist. It doesn't make sense

Reply:
Syntax also broken